server/core/cache.service.ts

```diff
@@ -0,0 +1,61 @@
+import { getAppSettings } from "./app.settings";
+
+interface Cache<T> {
+	[id: number]: T | null
+}
+
+export interface CacheStats {
+    cacheHits: number,
+    cacheMisses: number
+}
+
+const counts: CacheStats = {
```

Comment:
If I understood this correctly, this is a global cache hit/miss counter
Do we maybe want counters per instantiated caches?

Reply:
No need for now, this was mostly for debugging purposes.